eg/Classes/App.py

```diff
@@ -133,6 +133,7 @@ def OnExit(self):
             eg.PrintDebugNotice("The following threads did not terminate:")
             for thread in threads:
                 eg.PrintDebugNotice(" ", thread, thread.getName())
+
```

Comment:
I think this has nothing to do with the PR and should be left out or made in another PR for code cleanup. Same goes for other whitespace/blank line fixes